modules/cnv-template-blank-disk-datavolume.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-expanding-virtual-storage-with-blank-disk-images.adoc
+
+[id="cnv-template-blank-disk-datavolume_{context}"]
+= Template: DataVolume configuration file for blank disk images
+
+*blank-image-datavolume.yaml*
```

Comment:
I'd remove this line in favor of the descriptive sentence. (I don't see a sample yaml file that has a heading, and the yaml snippets with headings use the ".yaml title" format.)

Reply:
Ah, excellent catch! This was how our templates were in 1.4. The 2.0 file `cnv-template-vmi-pxe-config.adoc` has a heading: " = Template: virtual machine instance configuration file for PXE booting"

Unless you see an issue, I'll mirror the PXE config file since it's already merged.

Reply:
Ah, I think you should remove "*blank-image-datavolume.yaml*" but that you can wait and do that when you add descriptive intros to the modules. If you agree to take that as an improvement for the nearish future, I'll merge.

Reply:
@kalexand-rh Yes, let's plan to improve all of the template modules after this release.